Makefile.common

```diff
@@ -110,13 +112,17 @@ ARCH =  $(SIL)@mkdir -p $(@D); \
 	$(AR) $(ARFLAGS) $@ $^ 2>/dev/null
 
 LINK =  $(SIL)mkdir -p $(@D); \
-	(printf "LD  %-24s\n" "$(@F)"); \
+	(printf "LD  %-24s\n" "$(@:$(PREFIX_BUILD)/%=%)"); \
 	$(LD) $(LDFLAGS) -o "$@"  $^ $(LDLIBS)
 	
 HEADER = $(SIL)mkdir -p $(@D); \
 	(printf "HEADER %-24s\n" "$<"); \
 	cp -pR "$<" "$@"
 
+INSTALL_FS = $(SIL)(printf "INSTALL_FS %s\n" "$(@:$(PREFIX_FS)%=%)"); \
+	     mkdir -p "$(dir $@)"; \
+	     install -m 755 "$<" "$(dir $@)"
```

Comment:
use `"$(@D)"`.

Probably would be good to remove 4 spaces indent to make consistent with other variables here (I understand why You would put additional indent here tho)